src/PaprController.sol

```diff
@@ -226,7 +233,14 @@ contract PaprController is
             underlying.safeTransferFrom(msg.sender, params.swapFeeTo, amountIn * params.swapFeeBips / BIPS_ONE);
         }
 
-        _reduceDebt({account: account, asset: collateralAsset, burnFrom: msg.sender, amount: amountOut});
+        uint256 debt = _vaultInfo[account][collateralAsset].debt;
+        _reduceDebt({
```

Comment:
this is an interesting one we probably should have thought of--I'm not sure we're guaranteed amountOut? Need to add a test for this